app/components/shared/FriendlyTime.js

```diff
@@ -4,15 +4,21 @@ import friendlyRelativeTime from '../../lib/friendlyRelativeTime';
 
 class FriendlyTime extends React.Component {
   static propTypes = {
-    value: React.PropTypes.string.isRequired
+    value: React.PropTypes.string.isRequired,
+    capitalized: React.PropTypes.bool.isRequired
+  };
+
+  static defaultProps = {
+    capitalized: false
```

Comment:
I think this should be defaulted to `true` since that's what it was to start with?